src/index.html

```diff
@@ -2,16 +2,135 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            name="Surname"
+            id="surname"
+            autocomplete="off"
+          >
+        </div>
+        <div class="form-field">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            name="Name"
+            id="name"
+            autocomplete="off"
+          >
+        </div>
+        <div class="form-field">
+          <label for="old">How old are You?</label>
+          <input
+            type="number"
+            name="how_old"
+            value="12"
+            id="old"
+            min="1"
+            max="100"
+          >
+        </div>
+        <div class="form-field">
+          <label for="birt">Full date of birth:</label>
+          <input type="date" name="birth" id="birt">
```

Comment:
There shouldn't be more than 2 attributes in one line
```suggestion
          <input
            type="date"
            name="birth"
            id="birt"
          >
```
fix it everywhere and look through [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) one more time